schemas/spdx-schema.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema" : "http://json-schema.org/draft-07/schema#",
   "$id" : "http://spdx.org/rdf/terms",
-  "title" : "SPDX 2.2.2",
+  "title" : "SPDX 2.2",
```

Comment:
Did you mean to change this line?

Reply:
@seabass-labrax Yes - I'm including the minor version in the schema since it has changed since 2.2.1